### PR TITLE
Removed funny joke

### PR DESCRIPTION
**Pull request by DieGurke**
lol

### CODE REVIEW

File: src/main/java/envoy/server/data/User.java [hunk 1]
```diff
@@ -3,7 +3,17 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
```
Comment (CyB3RC0nN0R) [inline src/main/java/envoy/server/data/User.java]: These can be expressed with a wildcard import.

Comment (DieGurke) [reply src/main/java/envoy/server/data/User.java]: Done

**Comment by DieGurke** [comment]
!!

**Comment by CyB3RC0nN0R** [comment]
No.

**Comment by DieGurke** [comment]
You will approve these changes *fancy jedi hand movement*

**Pull request merged**